src/schemaToPropTypes.js

```diff
@@ -1,5 +1,5 @@
 let indentLevel = 1;
-const FILE_IMPORTS = `import PropTypes from 'prop-types';\n\n`;
+const FILE_IMPORTS = `/* eslint no-use-before-define: 0 */\nimport PropTypes from 'prop-types';\n`;
```

Comment:
I'd move this into its own const. The idea behind for me was to have all them as eventual possible options to be defined on a use-case basis… maybe using arguments in the terminal, or a txt file or something alike. How would you see this?

Reply:
Maybe even something like `FILE_HEADERS`, where we/users could add in there whatever they need, like other linters rules, or Authors, or whatever

Reply:
Ah I didn't see second comment, but yeah, I separated this a bit, naming i just maybe a bit different